less/forum/Post.less

```
@@ -185,6 +185,10 @@
   }
 }
 
+.Post--renderFailed {
+  background-color: @control-danger-color;
```

I think a less strong red would be better.

```suggestion
  background-color: @control-danger-bg;
```